src/server_manager/web_app/app.ts

```diff
@@ -876,9 +876,10 @@ export class App {
 
   private addAccessKey() {
```

Comment:
Let's use async-await for this whole function instead of mixing styles

Reply:
Done